runtime/onert/core/src/ir/LoweredGraph.cc

```diff
@@ -19,11 +19,11 @@
 #include <assert.h>
 #include <sstream>
 #include "util/logging.h"
-#include "pass/ConstantInsertionPass.h"
-#include "pass/ConstantLoweringPass.h"
-#include "pass/PermutationOperationPass.h"
-#include "pass/PermutationInsertionPass.h"
-#include "pass/PermutationEliminationPass.h"
+#include "compiler/pass/ConstantInsertionPass.h"
+#include "compiler/pass/ConstantLoweringPass.h"
+#include "compiler/pass/PermutationOperationPass.h"
+#include "compiler/pass/PermutationInsertionPass.h"
+#include "compiler/pass/PermutationEliminationPass.h"
```

Comment:
@wateret We need to divide `LoweredGraph` constructor's lowering routine and move into compiler later.

Reply:
I'm planning to move the whole `LoweredGraph` to compiler too 😄